db/schema.rb

```diff
@@ -318,7 +317,7 @@
     t.boolean "ready_for_status_check", default: false
     t.datetime "notification_sent_at", comment: "The time a notification was sent"
     t.datetime "last_batch_claimed_at"
-    t.string "sponsor_id", comment: "The identification number for USPS to recognize us and our flow (ex: Enhanced IPP)"
```

Comment:
I think this comment was intentionally added based on [review feedback](https://github.com/18F/identity-idp/pull/10759#discussion_r1628206348), so the sync-up should probably have been to include the comment in the migration itself.

cc @KeithNava @eileen-nava

Reply:
Good point. I suppose if it's important to have the comment, a separate migration just to add the comment might be the way to go.